alibi/explainers/cfrl_base.py

```diff
@@ -740,6 +740,9 @@ def _validate_target(Y_t: np.ndarray):
         Y_t
             Targets to be checked.
         """
+        if Y_t is None:
+            raise ValueError("Target can not be `None`.")
```

Comment:
Slightly confused, if `Y_t` can't be `None`, why is it allowed to be `Optional`?